www/js/app.js

```diff
@@ -358,7 +358,8 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
         return false;
     });
     $('#btnTop').on('click', function() {
-        $("#articleContent").contents().scrollTop(0);
+        let articleContent = document.getElementById('articleContent');
+        articleContent.contentWindow.scrollTo({top: 0,behavior: 'smooth'});
```

Comment:
@dheerajdlalwani This is still missing a space after comma.

Reply:
I really appreciate the fact that you are really particular about small things like these. These are the very things necessary to keep the codebase maintainable and make it easier for others to collaborate. I wish I was working under you! So much to learn from you :)

Reply:
Thanks @dheerajdlalwani , I'm just following the example of the people who have reviewed my code in the past!